### DISCUSSION TITLE
How can I hide bottom navigation bar on scroll of a LazyColumn inside a Tab

### DISCUSSION THREAD

**Discussion by shriharsha-srb** [adrielcafe/voyager#250] [General]
In [TabNavigationActiivity](https://github.com/adrielcafe/voyager/blob/main/samples/android/src/main/java/cafe/adriel/voyager/sample/tabNavigation/TabNavigationActivity.kt), say I want to hide/show bottom navigation bar based on scroll direction of a LazyColumn inside HomeTab, is it possible? How will i handle such scenario ?

**Comment by haitrvn** [comment]
https://developer.android.com/jetpack/compose/compositionlocal
Using CompositionLocal, you provide a state that handle hide/show bottom navigation, make sure that scope of CompositionLocal you provide must include bottom navigation, so bottom navigation observe and hide/show by that state, and LazyColumn changes according to the calculation based on state scrolling state

**Reply by amanverma-765** [reply]
could you please provide us a clear example of using CompositionLocal for this ?
